.swiftformat

```diff
@@ -1,5 +1,20 @@
 #Disable all and explicitly enable them to avoid unintentionally activating new rules in newer versions of SwiftFormat
---disable all
+#--disable all
```

Comment:
We want this to stay uncommented, reason described in the comment

Reply:
the `--rules` option only allows the [rules](https://github.com/nicklockwood/SwiftFormat#rules) you specify, so this is redundent?
```
And then individually enable just the rules you want. Alternatively, use the--rules argument to only enable the rules you specify:

--rules indent,linebreaks
```